src/components/Hero.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Box } from '@theme-ui/components'
+import heroBg from '../assets/images/hero-bg.jpg'
+import { graphql, useStaticQuery } from 'gatsby'
+import Image from 'gatsby-image'
+
+const Hero: React.FC = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      logo: file(relativePath: { eq: "images/logo.png" }) {
+        childImageSharp {
+          fluid(maxWidth: 1200) {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+      bandShot: file(relativePath: { eq: "images/band-shot.png" }) {
+        childImageSharp {
+          fluid(maxWidth: 1200) {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+    }
+  `)
+
+  return (
+    <Box
+      sx={{
+        background: `url(${heroBg}) no-repeat center`,
+        backgroundSize: 'cover',
+      }}
+    >
+      <Box
+        sx={{
+          maxWidth: 1200,
```

Comment:
Aren't you setting the maxWidth in the config.ts? pull in config so you don't get hung up if needed to change.

Reply:
Yup, good catch